modules/src/events.rs

```diff
@@ -140,14 +142,15 @@ pub fn get_all_events(result: RpcEvent) -> Result<Vec<IBCEvent>, String> {
 
             let actions_and_indices = extract_helper(&events)?;
             for action in actions_and_indices {
-                let ev = build_event(RawObject::new(
+                match build_event(RawObject::new(
                     height,
                     action.0,
                     action.1 as usize,
                     events.clone(),
-                ))
-                .map_err(|e| e.to_string())?;
-                vals.push(ev);
+                )) {
+                    Ok(ev) => vals.push(ev),
+                    Err(e) => warn!("error while building event {}", e.to_string()),
+                }
```

Comment:
Can we keep the `ev` variable (perhaps renamed to `event`) to improve readability a little bit?